### PR TITLE
update format API

### PR DESCRIPTION
**Pull request by iamkun**

### CODE REVIEW

File: src/locale/th.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import dayjs from 'dayjs'
 const locale = {
   name: 'th',
   weekdays: 'อาทิตย์_จันทร์_อังคาร_พุธ_พฤหัสบดี_ศุกร์_เสาร์'.split('_'),
-  weekdaysShort: 'อา._จ._อ._พ._พฤ._ศ._ส.'.split('_'),
+  weekdaysShort: 'อาทิตย์_จันทร์_อังคาร_พุธ_พฤหัส_ศุกร์_เสาร์'.split('_'),
```
Comment (iamkun) [inline src/locale/th.js]: @saknarak
In order to pass the unit test compared to moment.js, I've changed this locale a little bit. Is this still OK in thai?

Comment (saknarak) [reply src/locale/th.js]: Yes, it still OK in Thai.

```
// from moment.js
        weekdaysShort : 'อาทิตย์_จันทร์_อังคาร_พุธ_พฤหัส_ศุกร์_เสาร์'.split('_'),
        weekdaysMin : 'อา._จ._อ._พ._พฤ._ศ._ส.'.split('_'),
```

Comment (iamkun) [reply src/locale/th.js]: Thanks @saknarak , just wondering if we should add weekdaysMin as well?

**Pull request merged**